src/main/java/hudson/plugins/brakeman/BrakemanResult.java

```diff
@@ -20,6 +42,12 @@
     /** Unique identifier of this class. */
     private static final long serialVersionUID = -137460587767210579L;
 
+    private transient boolean useDeltaValues;
+    private transient Thresholds thresholds = new Thresholds();
+    /** The build history for the results of this plug-in. */
```

Comment:
why have the `useDeltaValues` and `threshold` fields.  it looks like you assign them a value in `evaluateStatus` but they are never read from?  pretty sure you can lose them.